src/neo4j_genai/filters.py

```diff
@@ -185,7 +185,7 @@ def _get_params_name(self):
         self._counter[key] += 1
         return param_name
 
-    def add(self, value):
+    def add(self, value: Union[str, int, list[Any]]) -> Union[str, int, list[Any]]:
```

Comment:
```suggestion
    def add(self, value: Any) -> str:
```

Reply:
It always returns the name of the parameter as string

Reply:
c9081d159d46cdaddbf90285271128c4cf7dc91a